lib/experimental/Overlays/Tooltip/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { Shortcut } from "@/experimental/Information/Shortcut"
+import { cn } from "@/lib/utils"
+import {
+  TooltipContent,
+  Tooltip as TooltipPrimitive,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/ui/tooltip"
+import React from "react"
+
+interface TooltipProps {
+  label: string
+  description?: string
+  children: React.ReactNode
+  shortcut?: string[]
```

Comment:
This feels a bit out of place, but at the same time I understand why we have it. 

Additionally, I'm worried that we need to put a bit more work into `shortcuts`, as they might be platform-dependant (windows vs mac) and might need better types.

We can leave this be as it's on `experimental` but I think we need to think deeper about it.

Reply:
Also, we should probably type this better. At the minimum least, I would delegate to whatever the `Shortcut` component accepts via `shortcut?: ComponentProps<typeof Shortcut>["keys"]`